routes/analysis_routes.py

```
@@ -1,4 +1,6 @@
-from flask import Blueprint, jsonify, Response,request, send_file
+from ctypes import c_buffer
```

Ruff (0.11.9)

`ctypes.c_buffer` imported but unused

Remove unused import: `ctypes.c_buffer`

(F401)